com.unity.sg2/Tests/Generation/GenerationTests.cs

```diff
@@ -43,9 +43,9 @@ public static void Setup()
             graph.RebuildContextData(propertyKey.Name, GetTarget(), "UniversalPipeline", "SurfaceDescription", true);
             //graph.RebuildContextData(contextKey.Name, GetTarget(),  "UniversalPipeline", "SurfaceDescription", false);
 
-            CPGraphDataProvider.GatherProviderCPIO(GetTarget(), out var descriptors);
-            foreach (var descriptor in descriptors)
-                LogDescriptor(descriptor);
+            //CPGraphDataProvider.GatherProviderCPIO(GetTarget(), out var descriptors);
```

Comment:
Do we think this code might come back or should it be deleted?

Reply:
Some engineers have found it useful to know how to get the information provided, but that could be done in a separate helper function somewhere (I have no strong opinions)